src/Microsoft.ML.HalLearners/OlsLinearRegression.cs

```diff
@@ -610,7 +610,7 @@ public IReadOnlyCollection<Double> TValues
         public IReadOnlyCollection<Double> PValues
         { get { return _pValues.AsReadOnly(); } }
 
-        internal OlsLinearRegressionPredictor(IHostEnvironment env, in VBuffer<float> weights, float bias,
+        public OlsLinearRegressionModelParameters(IHostEnvironment env, in VBuffer<float> weights, float bias,
             Double[] standardErrors, Double[] tValues, Double[] pValues, Double rSquared, Double rSquaredAdjusted)
```

Comment:
>Double[] standardErrors, Double[] tValues, Double[] pValues, Double rSquared, Double rSquaredAdjusted [](start = 12, length = 101)

can you make all this double arrays nullable by default?
so we can use 
`OlsLinearRegressionModelParameters(Host, in weights, bias);`
instead of 
`OlsLinearRegressionModelParameters(Host, in weights, bias, null, null, null, 1, float.NaN)`